docs/integrations/sources/surveymonkey.md

```diff
@@ -1,53 +1,53 @@
-# SurveyMonkey
+# Set up the SurveyMonkey Source Connector
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
-
-:::note
-
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
-
-:::
 
 <!-- env:oss -->
 
 ## Prerequisites
 
+**SurveyMonkey:**
+- Registered Private application [here](https://developer.surveymonkey.com/apps/)
+- SurveyMonkey authorization credentials
+- SurveyMonkey authorization method (oauth2.0)
+
```

Comment:
:::info
To learn more about setup and requirements associated with Private apps on SurveyMonkey, visit their documentation [here](https://api.surveymonkey.com/v3/docs#private-apps)
:::

Reply:
There are potential limits and account type requirements that we might not want to enumerate, but that we should guide users to be able to find easily.